x-pack/README.md

```diff
@@ -44,69 +44,76 @@ yarn test:server
 
 #### Running functional tests
 
-The functional tests are run against a live browser, Kibana, and Elasticsearch install. They build their own version of elasticsearch and x-pack-elasticsearch, run the builds automatically, startup the kibana server, and run the tests against them.
+The functional UI tests, the API integration tests, and the SAML API integration tests are all run against a live browser, Kibana, and Elasticsearch install. Each set of tests is specified with a unique config that describes how to start the Elasticsearch server, the Kibana server, and what tests to run against them. The sets of tests that exist today are *functional UI tests* ([specified by this config](test/functional/config.js)), *API integration tests* ([specified by this config](test/api_integration/config.js)), and *SAML API integration tests* ([specified by this config](test/saml_api_integration/config.js)).
+
+The script runs all sets of tests sequentially like so:
+* It builds Elasticsearch and x-pack-elasticsearch
```

Comment:
__nit:__ no more `x-pack-elasticsearch` (and `x-pack-kibana` below), right?

Reply:
Okay, replacing with `X-Pack`